runtime/tests/interpreter/resources_test.go

```diff
@@ -2344,7 +2347,8 @@ func TestInterpretOptionalResourceReference(t *testing.T) {
 	)
 
 	_, err := inter.Invoke("test")
-	require.NoError(t, err)
+	require.Error(t, err)
+	require.ErrorAs(t, err, &interpreter.InvalidatedResourceReferenceError{})
```

Comment:
`require.Error` is redundant here since `ErrorAs` already asserts that the error matches the target

```suggestion
	require.ErrorAs(t, err, &interpreter.InvalidatedResourceReferenceError{})
```

Reply:
hmm actually, now that master is merged to the stable cadence branch, `require.Error` should be replaced by `RequireError`, which also checks for error messages etc.